### PR TITLE
i18n update for German

### PR DESCRIPTION
**Pull request by nicokaiser**
This adds and improves some german translations

### CODE REVIEW

File: app/i18n/de/admin.php [hunk 1]
```diff
@@ -122,7 +122,7 @@
 		'title' => 'Erweiterungen',
 		'update' => 'Update verfügbar',
 		'user' => 'Benutzer-Erweiterungen',
-		'version' => 'Version',	// TODO - Translation
+		'version' => 'Version',
```
Comment (Frenzie) [inline app/i18n/de/admin.php:125]: Did you also add it to the ignore list? :-)

**Review by Frenzie — APPROVED** [review]
lgtm

File: app/i18n/de/admin.php (hunk 1 above)
Comment (nicokaiser) [reply app/i18n/de/admin.php:125]: Not yet, I'll update the ignore list.

File: cli/i18n/ignore/de.php [hunk 2]
```diff
@@ -1,6 +1,66 @@
...
+	'admin.extensions.name',
+	'admin.extensions.version',
+	'admin.stats.feed',
+	'admin.user.admin',
+	'conf.sharing.blogotext',
+	'conf.sharing.diaspora',
+	'conf.sharing.facebook',
+	'conf.sharing.shaarli',
+	'conf.sharing.twitter',
+	'conf.sharing.twallabag',
```
Comment (Frenzie) [inline cli/i18n/ignore/de.php]:
```suggestion
	'conf.sharing.wallabag',
```

File: cli/i18n/ignore/de.php [hunk 3]
```diff
@@ -1,6 +1,66 @@
...
+	'gen.share.wallabagv2',
+	'index.about.version',
+	'index.about.website',
+	'install.bdd.host',
+	'install.title',
+	'sub.api.title',
+	'sub.bookmarklet.title',
+	'sub.category.information',
+	'sub.feed.information',
+	'sub.tag.name'
```
Comment (Frenzie) [inline cli/i18n/ignore/de.php]:
```suggestion
	'sub.tag.name',
```

File: cli/i18n/ignore/de.php [hunk 4]
```diff
@@ -1,6 +1,66 @@
...
+	'gen.lang.he',
+	'gen.lang.it',
+	'gen.lang.kr',
+	'gen.lang.nl',
+	'gen.lang.oc',
+	'gen.lang.pt-br',
+	'gen.lang.ru',
+	'gen.lang.sk',
+	'gen.lang.tr',
+	'gen.lang.zn-cn',
```
Comment (Frenzie) [inline cli/i18n/ignore/de.php]:
```suggestion
	'gen.lang.zh-cn',
```

**Review by Frenzie — COMMENTED** [review]
I think you may still be missing a few, please see https://travis-ci.org/github/FreshRSS/FreshRSS/jobs/725875996

**Pull request merged**